### ISSUE TITLE
Fails to get past Cloudflare

### ISSUE THREAD

**Issue by ekkis** [stealify/request-cloudflare#14]
I tried the following script:
```js
const { requestCloudflare } = require('request-cloudflare')
 const fetch = requestCloudflare.promises.get
 fetch('https://bank.codes').then(console.log).catch(console.log)
```
and what I get is what currently get from curl, a challenge.  anything one can do?

**Comment by frank-dspeed** [comment]
Yes we can do something is the short answer the longer answer is we have no one who will do it at present. I am total alone while i have tons of knowledge i have no people that help me to bring the code in shape. But i readed your investor if you have a real interest to get a good solution for this we could talk a bit i have a lot of news for you that are partial related to your issue.

I am at present working on something that i call Chromium Virtual Environments it is a massive project but i could put priority on your exact use case in general it opens a total new way to interact with web pages and the web as a whole.

and it enables to solve captchas via image processing. As also it would allow to apply advanced logic on request to circumvent the captures and cache the working methods to circumvent it . Lets call it reverse engineering the ddos settings.

But as sayed we should maybe talk in general maybe you should send me a email to frank+ekkis@lemanschik.com i have also good news for your crypto stuff i guess i have a lot of solutions and with some one like you who understands them we could pull of the next google / microsoft without inventing new low level apis or code only logic is needed and a small team that executes.

I did in fact over 30 years of research to pull that of in a simple consistent way with confidence

**Comment by ekkis** [comment]
I've ended up using Pupeteer, which seems to work well.  I published https://github.com/ekkis/bank-codes on the basis of that.  but I'm open to a chat if you are, maybe we can find common ground to work together

**Comment by lemanschik** [comment]
@ekkis puppeteer is a step into the right direction i wanted to suggest you as incremental step to use the free tier of browserless.io it is puppeteer in docker but your right we should in general work together i need at last people who understand the problems.

**Comment by ekkis** [comment]
you can find me on Telegram under the same rubrik
